file_paths.md

```diff
@@ -339,6 +340,39 @@ The package will install a service named `puppetmaster`, create a
         puppetmaster *                    # writeable by puppetmaster
             puppetmaster.pid
 
+# pxp-agent
+
+This package executes Puppet and replies to PXP requests.
+
+## *nix
+
+    /opt/puppetlabs/pxp-agent *
+        spool                             # :spooldir
+        modules *                         # :modulesdir
+            puppet *                      # executes Puppet and provides results of Puppet runs
+
+    /var/log/puppetlabs/pxp-agent *       # :logdir
+        pxp-agent.log                     # enabled by default
+
+    /etc/puppetlabs/pxp-agent *
+        pxp-agent.cfg                     # :configfile
+        modules.d *                       # :modules_configdir
+            puppet.cfg *                  # configuration file of the `puppet` module
+
+## Windows
+
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        var *
+            spool                         # :spooldir
+            modules *                     # :modulesdir
+                puppet *                  # executes Puppet and provides results of Puppet runs
+            logs *                        # :logdir
+                pxp-agent.log             # enabled by default
```

Comment:
Nevermind, the font made this look uppercase to me. Ugh. Sorry. Nothing to see here.